skate-plugin/build.gradle.kts

```diff
@@ -9,6 +9,10 @@ group = "com.slack.sgp.intellij"
 version = "1.0-SNAPSHOT"
 
 repositories { mavenCentral() }
+java {
+    sourceCompatibility = JavaVersion.VERSION_20
+    targetCompatibility = JavaVersion.VERSION_20
+}
```

Comment:
this shouldn't be necessary, do you know why you needed to add it?

Reply:
I can probably remove it because I had adapted a java file from a tool window sample that originally was in Java, and when I copied it over initially there was an error that required me to add this.